src/controllers/notification/subscribe.js

```diff
@@ -0,0 +1,36 @@
+const models= require('../../models');
+import { decode } from 'jsonwebtoken';
+import getDataFromToken from '../../helper/tokenToData';
+
+import hasSubscribed from '../../services/subscriptions';
+
+const subscribe= async (req,res, next)=>{
+    const decoded= await getDataFromToken(req,res,next);
+    const subscription= req.body;
+
+    // console.log('@@@@@@@@@hasSubscribed: ', await hasSubscribed(subscription.keys.auth));
```

Comment:
@Noel2002 kindly remove console.logs codes and any other commented & unused code.

Reply:
@bdushimi yeah! I will remove those.